packages/frontend/src/components/EmptyFlows/index.tsx

```diff
@@ -35,57 +36,63 @@ const flowTemplates: FlowTemplateProps[] = [
 ]
 
 interface EmptyFlowsProps {
-  CreateFlowLink: As
   count?: number
 }
 
 export default function EmptyFlows(props: EmptyFlowsProps) {
-  const { CreateFlowLink, count } = props
+  const { count } = props
```

Comment:
nit: you can destructure this directly in args e.g.
```
export default function EmptyFlows({ count }: EmptyFlowsProps) {
```

Reply:
sorz i just very used to declaring it in the function cus it looks clearer to me